admin_guide/high_availability.adoc

```diff
@@ -153,29 +169,35 @@ $ ping hello-openshift.shard1.v3.rhcloud.com
 [[configuring-ip-failover]]
 == Configuring IP Failover
 
-Using IP failover involves switching IP addresses to a redundant or stand-by
-set of nodes on failure conditions.
+Use the `oadm ipfailover` command with suitable xref:../admin_guide/high_availability.adoc#options-environment-variables[options], to create ipfailover deployment configuration.
 
 [IMPORTANT]
 ====
-At this time of writing, ipfailover is not compatible with cloud
-infrastructures. In the case of AWS, an Elastic Load Balancer (ELB) can be used
-to make {product-title} highly available,
+Currently, ipfailover is not compatible with cloud
+infrastructures. For AWS, an Elastic Load Balancer (ELB) can be used to make {product-title} highly available,
 link:http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/elb-getting-started.html[using
 the AWS console].
 ====
 
-The `oadm ipfailover` command helps set up the VIP failover configuration. As
-an administrator, you can configure IP failover on an entire cluster, or on a
-subset of nodes, as defined by the labeled selector. If you are running in
-production, match the labeled selector with at least two nodes to ensure you
-have failover protection and provide a `--replicas=<n>` value that matches the
-number of nodes for the given labeled selector:
+As an administrator, you can configure ipfailover on an entire cluster, or on a subset of nodes, as defined by the label selector. You can also configure multiple IP failover deployment configrations in your cluster,  where each one is independent of the others. The `oadm ipfailover` command creates a ipfailover deployment configration which ensures that a failover pod runs on each of the nodes matching the constraints or the label used. This pod runs link:http://www.keepalived.org/[*Keepalived*] which uses VRRP (Virtual Router Redundancy Protocol) among all the *Keepalived* daemons to ensure that the service on the watched port is available, and if it is not, *Keepalived* will automatically float the VIPs.
+
+Use a `--selector=<label>` with at least two nodes to select
+the nodes. Also, set a `--replicas=<n>` value that matches the
+number of nodes for the given labeled selector.
+
+The `oadm ipfailover` command includes command line options that set environment
+variables that control *Keepalived*. The
+xref:../admin_guide/high_availability.adoc#options-environment-variables[environment variables]
+start with OPENSHIFT_HA_ and they can be changed as needed.
```

Comment:
OPENSHIFT_HA_ should perhaps be markuped differently. Maybe:

`OPENSHIFT_HA_*`

(I'm presuming that because it "starts with" that there's an ending to the parameter that changes)